src/extension/content-script/liquid.js

```diff
@@ -66,6 +69,10 @@ async function init() {
         origin: getOriginData(),
       };
 
+      if (!account.hasMnemonic) {
+        messageWithOrigin.action = `public/liquid/providerOnboard`;
```

Comment:
can you add a comment here? so it overrides the action so the user can go through onboarding to setup their master key. Then they need to refresh this page to do what they were going to do before, right?